src/rust-2021/disjoint-capture-in-closures.md

```diff
@@ -3,7 +3,7 @@
 ## Summary
 
 - `|| a.x + 1` now captures only `a.x` instead of `a`.
-- This can subtly change the drop order of things.
+- This can subtly change the drop order of things, and auto traits usage with closures.
```

Comment:
I'm not sure this is very clear. Maybe the following would be better?
```suggestion
- This can subtly change the drop order of things and whether auto traits are applied to closures or not.
```